scripts/hca-utils/hca_utils/utils.py

```diff
@@ -133,28 +132,19 @@ def populate_row_id_csv(row_ids: Set[str], temp_file):
 
         sd_writer.writerows([[rid] for rid in row_ids])
 
-    @staticmethod
-    def delete_csv(filename: str):
-        """
-        Remove the csv after it has been uploaded so that there isn't a deluge of csv files in the local directory.
-        :param filename: The filename of the file to delete.
-        :return:
-        """
-        os.remove(filename)
-
     # gcs (cloud storage) interactions
-    def put_csv_in_bucket(self, local_filename: str, target_table: str) -> str:
+    def put_csv_in_bucket(self, local_file, target_table: str) -> str:
         """
         Puts a local file into a GCS bucket accessible by Jade.
-        :param local_filename: The filename of the file to upload.
-        :param target_table: The table name with which to fomrat the target filename.
+        :param local_file: The file to upload.
+        :param target_table: The table name with which to format the target filename.
         :return: The gs-path of the uploaded file.
         """
         storage_client = storage.Client(project=self.bucket_project, credentials=self.gcp_creds)
         bucket = storage_client.bucket(self.bucket)
-        blob = bucket.blob(local_filename)
-        blob.upload_from_filename(local_filename)
         target_filename = self._format_filename(table=target_table)
+        blob = bucket.blob(target_filename)
+        blob.upload_from_file(local_file, rewind=True)
```

Comment:
❓ What does `rewind=True` mean here?

Reply:
ah we put this here when there was a single file open operation, and we wanted to reset the file marker (idk the correct term) back to the beginning of the file; guess it wouldn't be necessary now since we open it in a separate operation to read/upload. Can remove.